Newer setuptools versions do not support the `use_2to3` argument anymore; remove
this from setup.py and run the conversion in the post-patch phase instead. Additionally,
use `setup` from the `setuptools` package and not upstream's patched version..
--- setup.py.orig	2022-02-23 21:44:19.000000000 -0500
+++ setup.py	2022-02-23 21:45:03.000000000 -0500
@@ -13,7 +13,7 @@
         from distribute_setup import use_setuptools
         use_setuptools()

-    extra = {'use_2to3': True,
+    extra = {'use_2to3': False,
              'test_dirs': test_dirs,
              'test_build_dir': 'build/tests',
              'pyversion_patching': True,
@@ -22,8 +22,7 @@
     extra = {}

 try:
-    from setup3lib import setup
-    from setuptools import find_packages
+    from setuptools import find_packages, setup
     addl_args = dict(
         zip_safe = False,
         packages = find_packages(),
